src/Chapter2.hs

```diff
@@ -741,8 +762,12 @@ the list with only those lists that contain a passed element.
 
 🕯 HINT: Use the 'elem' function to check whether an element belongs to a list
 -}
-contains = error "contains: Not implemented!"
-
+contains :: Int -> [[Int]] -> [[Int]]
+contains z [] = []
+contains z [x] = [x | elem z x]
+contains z (x:xs)
+    | z `elem` x = x : contains z xs
+    | otherwise = contains z xs
```

Comment:
You've correctly used `elem` to check whether the element is in the list! You can go one step further and use standard function `filter` instead of manual recursion and pattern-matching.